src/ucp/rma/rma_send.c

```diff
@@ -215,6 +237,11 @@ ucs_status_t ucp_put_nbi(ucp_ep_h ep, const void *buffer, size_t length,
 
     /* Fast path for a single short message */
     if (ucs_likely((ssize_t)length <= (int)rkey->cache.max_put_short)) {
+        status = ucp_rma_check_mem_type(ep, buffer, length);
```

Comment:
can we set max_short to -1 in case memtype MDs are detected?
just like for tag_send

Reply:
done